### PR TITLE
Update to asp.net 2.1

### PR DESCRIPTION
**Pull request by mdallago**

### CODE REVIEW

**Comment by mdallago** [comment]
It's not ready yet. We need to wait the support of Appveyor and travis before merge

**Comment by mdallago** [comment]
Travis is building Ok

**Comment by mdallago** [comment]
Appveyor is building ok

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ branches:
   only:
   - master
   - dev
+  - feature/RA-98
```
Comment (dario-ramos) [inline .travis.yml]: Remove branch from build before merging

**Pull request merged**